dev_apps/music-nga/js/app.js

```diff
@@ -22,9 +22,20 @@ perfMark.log = () => Object.keys(perfMark).forEach((marker) => {
 // exists in the DOM and is marked as ready to be displayed.
 perfMark('navigationLoaded');
 
+const VIEWS = {
+  ALBUM_DETAIL:    { TAB: 'albums',    URL: '/views/album-detail/index.html' },
```

Comment:
I'd lower-case these attributes; I don't think they need to be capitalized, since they're not consts themselves (just members of a const).